pkgs/stdenv/generic/check-meta.nix

```diff
@@ -53,7 +53,7 @@ let
     hasLicense attrs &&
     isUnfree (lib.lists.toList attrs.meta.license);
 
-  isMarkedBroken = attrs: attrs.meta.broken or false;
+  isMarkedBroken = attrs: attrs.meta.broken or false != false;
```

Comment:
```suggestion
  isMarkedBroken = attrs: attrs.meta.broken or true;
```

Reply:
This change would mark all packages without a `meta.broken` attribute as broken.

The original `meta.broken or false != false` is correct, because the `or` is binding stronger than the `!=`.